src/fragments/attributes.ts

```diff
@@ -25,9 +34,7 @@ export const attributeDetailsFragment = gql`
     storefrontSearchPosition
     valueRequired
     values {
-      id
-      name
-      slug
+      ...AttributeValueFragment
       type
```

Comment:
Shouldnt type also be here?

Reply:
Actually, `type` is a deprecated field, so it shouldn't be anywhere. I have made an investigation and I have found it is not even used anywhere, so I'll remove it completely.

Reply:
done